api_tests/preprint_providers/views/test_preprint_provider_detail.py

```diff
@@ -26,11 +26,20 @@ def preprint_provider_two(self):
     def fake_url(self):
         return '/{}preprint_providers/fake/'.format(API_BASE)
 
+    @pytest.fixture()
+    def fake_url_generalized(self):
```

Comment:
I think it might be better for the generalized to be the default name and the old route to be named _deprecated. 
so provider_url is the better way to do it, deprecated is the old way.